### ISSUE TITLE
Only one of multiple cookies deleted with "max-age=0" are synced

### ISSUE THREAD

**Issue by saschbro** [DevExpress/testcafe-hammerhead#2828]
### What is your Scenario?

I reported the following issue on StackOverflow which was fixed with `28.2.5` by @Aleksey28:
https://stackoverflow.com/questions/74684772/deleted-cookies-arent-deleted-properly-and-still-sent-with-requests

However, in my use case multiple cookies are deleted in the client via `document.cookie = "<cookie-name>=; path=/; max-age=0;"`, but now only the first cookie is really synced. If I set `max-age=1` instead all cookies are synced.

I think this is because of a "timing" issue in the `syncCookie` function. If I add `&& maxAge > 0` to the "if statement" in `src/client/sandbox/cookie/index.ts::142` all cookies are synced with `max-age=0`.

### What is the Current behavior?

Only one of multiple cookies set via `document.cookie = "cookieX=; path=/; max-age=0;"` (with X = 1, .., n) is synced.

### What is the Expected behavior?

All cookies set via `document.cookie = "cookieX=; path=/; max-age=0;"` (with X = 1, .., n) are synced.

### What is your public website URL? (or attach your complete example)

Please see my example that uses TestCafe `2.2.0-rc.1` which uses Hammerhead `28.2.5`:

[testcafe-delete-cookies.zip](https://github.com/DevExpress/testcafe-hammerhead/files/10293214/testcafe-delete-cookies.zip)

#### deleteCookies.test.js
```
import { Selector } from "testcafe";

fixture`Delete cookies`.page`http://localhost:8080/index.html`;

test(`Cookies to delete are synced`, async (t) => {
  await t.debug();

  const buttonMaxAge0 = Selector("#maxAge0");
  // should sync 20 cookies but only syncs one
  await t.click(buttonMaxAge0);

  const buttonMaxAge1 = Selector("#maxAge1");
  // syncs 20 cookies as expected
  await t.click(buttonMaxAge1);

  await t.wait(10000);
}).disablePageCaching;
```
#### index.html:
```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="UTF-8" />
    <title>Delete Multiple Cookies</title>
  </head>
  <body>
    <button id="maxAge0">Set cookies with max-age 0</button>
    <button id="maxAge1">Set cookies with max-age 1</button>
    <script>
      document.getElementById("maxAge0").addEventListener("click", function () {
        for (let i = 0; i < 20; i++) {
          document.cookie = `cookie${i}=; path=/; max-age=0;`;
        }
      });

      document.getElementById("maxAge1").addEventListener("click", function () {
        for (let i = 0; i < 20; i++) {
          document.cookie = `cookie${i}=; path=/; max-age=1;`;
        }
      });
    </script>
  </body>
</html>
```

### Steps to Reproduce

1. `npm ci`
1. `npm run serve` in one terminal
1. `npm test` in second terminal
1. Open developer console (F12)
   1. Open Application > Cookies
1. Step through test using "Next Action"
1. For `max-age=0` only 1 cookie is synced
1. For `max-age=1` all 20 cookies are synced

### TestCafe version

2.2.0-rc.1

### Node.js version

16.19.0

### Command-line arguments

testcafe chrome test/deleteCookies.test.js

**Comment by Aleksey28** [comment]
Hi @saschbro 

There is a [PR with a fix](https://github.com/DevExpress/testcafe-hammerhead/pull/2830). It will be merged and published in the nearest release after 2.2.0.

**Comment by saschbro** [comment]
Hi @Aleksey28,

Perfect, thanks a lot! I'll re-test when it's released.
